lib/rucio/daemons/rsedecommissioner/profiles/types.py

```diff
@@ -0,0 +1,75 @@
+"""Types used for profile definitions."""
```

Comment:
Need a full header like in the other files.

```
Header is incorrect in file lib/rucio/daemons/rsedecommissioner/profiles/types.py
```

Reply:
Header added.